moped-editor/src/views/projects/projectView/ProjectView.js

```diff
@@ -457,22 +455,48 @@ const ProjectView = () => {
                           </Breadcrumbs>
                         </Box>
                       </Grid>
-                      <Grid item xs={1} md={1}>
-                        <Box
-                          className={classes.followDiv}
-                          onClick={() => handleFollowProject()}
-                        >
-                          <Tooltip title={isFollowing ? "Unfollow" : "Follow"}>
-                            {isFollowing ? (
-                              <BookmarkIcon className={classes.unfollowIcon} />
-                            ) : (
-                              <BookmarkBorderIcon
-                                className={classes.followIcon}
-                              />
-                            )}
-                          </Tooltip>
-                        </Box>
-                      </Grid>
+                      {!isProjectDeleted && (
```

Comment:
I am conditionally rendering the follow icon based on if a project is deleted or not because I think the banner looks better without the follow icon above it.